chromium_src/crypto/obsolete/md5.h

```diff
@@ -6,6 +6,10 @@
 #ifndef BRAVE_CHROMIUM_SRC_CRYPTO_OBSOLETE_MD5_H_
 #define BRAVE_CHROMIUM_SRC_CRYPTO_OBSOLETE_MD5_H_
 
+// In order to continue using MD5 (which has now moved to //crypto/obsolete) for
```

Comment:
Can we create a follow-up issue here to use sha or something instead of md5? This whole thing seems odd, isn't there chromium code that handles this for us @simonhong ?

Reply:
> but it sets as default browser via system UI

I don't think it makes sense to maintain this code to do something different than chromium here, but let's create a follow-up issue to look at it